### DISCUSSION TITLE
Add/edit milestones

### DISCUSSION THREAD

**Discussion by KevinRaney** [community/community#11095] [Mobile]
It would be nice if we could add/edit milestones within the mobile app. 

Preferably, if we can make new milestones on the fly when adding/editing an issue.

**Comment by mxie** [comment]
Thanks for your feedback @KevinRaney! Support for milestone management is currently not on our roadmap but we're open to exploring this feature if there's a high demand. We'll post updates to this thread if anything changes.

**Reply by jsoref** [reply]
How much demand do you need?

I'm exploring using GitHub mobile and this was one of the first shortcomings I tripped on.

**Reply by KevinRaney** [reply]
I agree I found it pretty fast myself, it makes using the mobile app cumbersome and takes me away from what I'm doing in that moment to go and add a new milestone/label. Hopefully they will implement it soon as more people bump this issue.
